website/client/src/components/snackbars/notifications.vue

```diff
@@ -19,40 +29,294 @@
     width: 350px;
     z-index: 1400; // 1400 is above modal backgrounds
 
-    &-top-pos {
-      &-normal {
-        top: 65px;
-      }
+    top: var(--current-scrollY);
 
-      &-sleeping {
-        top: 105px;
-      }
-    }
+    justify-content: flex-end;
+    display: flex;
+  }
+
+  .animations-holder {
+    position: relative;
+    display: block;
+  }
+
+  .notification-item {
+    transition: transform 0.25s ease-in, opacity 0.25s ease-in;
+  }
+
+  .notifications-move {
+    // transition: transform .5s;
+  }
+
+  .notifications-enter-active {
+    // transition: opacity .5s;
+  }
+
+  .notifications-leave-active {
+    position: absolute;
+    right: 0;
+  }
+
+  .notifications-enter,
+  .notifications-leave-to {
+    opacity: 0;
+    right: 0;
   }
 </style>
 
 <script>
 import { mapState } from '@/libs/store';
 import notification from './notification';
+import { sleepAsync } from '../../../../common/script/libs/sleepAsync';
+
+const notificationsVisibleAtOnce = 4;
+const removalInterval = 2500;
+const delayDeleteAndNew = 60;
+const delayFilling = 240;
 
 export default {
   components: {
     notification,
   },
+  props: {
+    preventQueue: {
+      type: Boolean,
+      default: false,
+    },
+    debugMode: {
+      type: Boolean,
+      default: false,
+    },
```

Comment:
Do we need to keep the debug functionality in all the way to production?

Reply:
sadly there aren't any compiler flags that could remove it on production...  (atleast I don't know any, but if there are we could use something like this?)

I think it should stay (even if its only enabled in storybook) the amount of how often it had to be debugged - these code / debug-logs helped alot to speed things up - otherwise me or someone else who needs to debug it for (hopefully no) other issues it would speed up to get all needed logs in place instead of going through again and adding them again and again

(thats why the property to enable it)